tools/docker/scripts/keycloak-setup.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+echo running > /startup/status
```

Comment:
@Alex-Izquierdo I am using the Keycloak-setup script in minikube which won't have the volumes mounted. Should I copy this script into minikube directory.

Reply:
@Alex-Izquierdo Isn't the wget good enough for this shared script. It will wait till Keycloak's url is accessible.

Reply:
@mkanoor 
With my latest change minikube works as well. I tested both.